tests/test_fft.py

```diff
@@ -3,58 +3,57 @@
 import numpy as np
 
 from aspire import config
-from aspire.numeric import fft_object
+from aspire.numeric import fft_object, numeric_object
 
 fft_backends = ["scipy", "pyfftw"]
-numeric_modules = [np, np]
+numeric_modules = ["numpy", "numpy"]
```

Comment:
re `"numpy", "numpy"` If the code requires matching 2-tuples (it does, zip) then this should either be notated, the tuples constructed, or if it would work just take product of numeric X fft (maybe one of those combo's is not sane, don't recall).

If the code is otherwise good to go, a developer note regarding the implicit structure would be fine.

Reply:
Agreed. Let's make these lists of tuples (both for the `*_backends` and `*_objects` variables), that way the dependency is explicit.

Reply:
Used list of tuples in commit 7358c11.

Reply:
Cool. Looks like the second should be `("numpy", "pyfftw")`, though.

Reply:
Thanks. Fixed the typo and rebased it in commit 2e20372.